### DISCUSSION TITLE
Add Screen name to the members list

### DISCUSSION THREAD

**Discussion by shbchk** [ExpressionEngine/ExpressionEngine#3125] [Feature Requests]
I'm using email-as-username on [my website](https://vurchel.com/). 

In members list I see this:

```
|-------------------------
| ID   email1@email.email
|      email1@email.email
|-------------------------
| ID   email2@email.email
|      email2@email.email
|-------------------------
| ID   email3@email.email
|      email3@email.email
|-------------------------
```
![Screenshot 2023-03-15 194448](https://user-images.githubusercontent.com/12234507/225364004-13690354-a2a7-44ee-bc7a-35d3bed1fe48.png)

Maybe we can check `if (email === username) {show Screen name instead of Username}` or just add Screen name there?

**Comment by shbchk** [comment]
BTW, Screen name is also needed in the list because it appears here:

![Screenshot 2023-03-15 194742](https://user-images.githubusercontent.com/12234507/225366046-c50da792-d80e-4995-8204-f10294bca310.png)

**Comment by chunkitdesign** [comment]
You can change the username to screenname under Account>Username & Password. 
[Screenshot ee7](https://www.dropbox.com/s/82safmpd9gpkvdu/CleanShot%202023-03-15%20at%2015.49.14%402x.png?dl=0)

**Comment by intoeetive** [comment]
@TomJaeger what would you say?

I think it's valid request and we need to list all of email + username + screen name here

**Comment by TomJaeger** [comment]
Just to make sure I'm following, this is adding screen name to the members listing screen "all members" in screen shot above.. but would also apply to the members sorted by a give role?

I would be fine with adding this, however I view it as a bit of a short term solution... My hesitation is that we likely want to head more in the direction of the entries manager, and file manager allowing people to manually set the columns.  Specifically as we continue to open the members section to additional fields.

**Reply by intoeetive** [reply]
I am looking into this as temporary solution as well, until we implement entry-manager-style listing of members.

And I would think it needs to apply to all member listing pages - both full and filtered by role
